manifests/agent.pp

```diff
@@ -275,15 +276,18 @@
   case $::kernel {
     'Linux': {
       if $manage_repo {
-        class { 'wazuh::repo': }
+        class { 'wazuh::repo':
+          yumrepo_enabled => $yumrepo_enabled,
+        }
         if $::osfamily == 'Debian' {
           Class['wazuh::repo'] -> Class['apt::update'] -> Package[$agent_package_name]
         } else {
           Class['wazuh::repo'] -> Package[$agent_package_name]
         }
       }
       package { $agent_package_name:
-        ensure => "${agent_package_version}-${agent_package_revision}", # lint:ignore:security_package_pinned_version
+        ensure          => "${agent_package_version}-${agent_package_revision}", # lint:ignore:security_package_pinned_version
+        install_options => [{ '--enablerepo'  => 'wazuh' }],
```

Comment:
This parameter breaks the installation for Debian or Ubuntu systems:

E: The command line option --enablerepo=wazuh is not understood in combination with the other options

We need that this parameter is not added to these systems